libs/sdk-model/src/execution/buckets/index.ts

```diff
@@ -423,3 +424,26 @@ export function disableComputeRatio<T extends IAttributeOrMeasure>(item: T): T {
 
     return item;
 }
+
+/**
+ * Describes the type of the function used to modify the bucket items.
+ *
+ * @public
+ */
+export type BucketItemModifier = (bucketItem: IAttributeOrMeasure) => IAttributeOrMeasure;
```

Comment:
please rename to `BucketItemModification` (we use similar elsewhere.. MeasureModifications function etc)

Reply:
Thanks, I've updated the PR.